BUILD.bazel

```diff
@@ -0,0 +1,68 @@
+package(default_visibility = ["//visibility:private"])
+
+genrule(
+    name = "re2c_config",
+    srcs = [
+        "CMakeLists.txt",
+        "//bazel:re2c_config.h",
+    ],
+    outs = ["config.h"],
+    cmd = """
+        cat $(location //:CMakeLists.txt) |\\
+            sed -rne 's/project\\(re2c VERSION ([^ ]+).*/#define PACKAGE_VERSION "\\1" RE2C_VERSION_SUFFIX/p' >> "$@"
+        cat $(location //bazel:re2c_config.h) >> "$@"
+    """,
+)
+
+cc_library(
+    name = "re2c_config_cc",
+    hdrs = ["config.h"],
+    defines = ["""RE2C_STDLIB_DIR='"$(BINDIR)/re2c.runfiles/re2c/include"'"""],
```

Comment:
Are the single quotes needed? How does expansion work in this case?

Reply:
The single quotes enclose the string that will end up as the definition. So here we get a definition with double quotes which then become a string. Given that this holds a directory there could be spaces and so having the double quotes already here is the best option.